langchain4j-vertex/src/test/java/dev/langchain4j/model/vertex/VertexAiChatModelIT.java

```diff
@@ -0,0 +1,32 @@
+package dev.langchain4j.model.vertex;
+
+import dev.langchain4j.data.message.AiMessage;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class VertexAiChatModelIT {
+
+    @Test
+    @Disabled("To run this test, you must have provide your own endpoint, project and location")
+    void testChatModel() {
+        VertexAiChatModel vertexAiChatModel = new VertexAiChatModel(
```

Comment:
pls use builder, not clear what all those parameters are

Reply:
ok, changed